### PR TITLE
Fix broken link in quickstart & tutorials

### PR DESCRIPTION
**Pull request by Abinanthan47**
Fixed broken link in README.md

Quickstart & Tutorials

[Quickstart: Chatbot:](https://docs.copilotkit.ai/quickstart-chatbot?ref=github_readme) 

From : https://docs.copilotkit.ai/quickstart-chatbot?ref=github_readme
Screenshot 2024-10-05 102609

To  : https://docs.copilotkit.ai/quickstart

Screenshot 2024-10-05 102717

[Tutorial: Textarea Autocomplete:](https://docs.copilotkit.ai/tutorial-textarea/overview?ref=github_readme) 

From : https://docs.copilotkit.ai/tutorial-textarea/overview?ref=github_readme
Screenshot 2024-10-05 102908

To :  https://docs.copilotkit.ai/tutorials/ai-powered-textarea/overview
Screenshot 2024-10-05 103024

### CODE REVIEW

**Review by NathanTarbert — APPROVED** [review]

**Comment by NathanTarbert** [comment]
Thank you @Abinanthan47!

File: README.md [hunk 1]
```diff
@@ -82,9 +82,9 @@
 
 There are several easy ways to get started with CopilotKit:
 
-- [**Quickstart: Chatbot:**](https://docs.copilotkit.ai/quickstart-chatbot?ref=github_readme) In just two minutes, add an AI Chatbot to your app with the ability to read application state and take actions.
+- [**Quickstart: Chatbot:**](https://docs.copilotkit.ai/quickstart) In just two minutes, add an AI Chatbot to your app with the ability to read application state and take actions.
```
Comment (NathanTarbert) [inline README.md:85]: link should be - https://docs.copilotkit.ai/quickstart?ref=github_readme

File: README.md [hunk 2]
```diff
@@ -82,9 +82,9 @@
...
 - [**Tutorial: Todo List Copilot:**](https://docs.copilotkit.ai/tutorial-ai-todo-list-copilot/overview?ref=github_readme) For a deeper dive into CopilotKit, take a simple todo list app and supercharge it with an AI chat popup.
-- [**Tutorial: Textarea Autocomplete:**](https://docs.copilotkit.ai/tutorial-textarea/overview?ref=github_readme) For a deeper dive into CopilotKit, we'll take a simple email client app and add an AI-powered textarea with autocompletions and AI insertions/edits.
+- [**Tutorial: Textarea Autocomplete:**](https://docs.copilotkit.ai/tutorials/ai-powered-textarea/overview) For a deeper dive into CopilotKit, we'll take a simple email client app and add an AI-powered textarea with autocompletions and AI insertions/edits.
```
Comment (NathanTarbert) [inline README.md:87]: Link should be - https://docs.copilotkit.ai/tutorials/ai-powered-textarea/overview?ref=github_readme

**Review by NathanTarbert — COMMENTED** [review]
Please make these changes to the links @Abinanthan47.

File: README.md (hunk 1 above)
Comment (Abinanthan47) [reply README.md:85]: ok i will

**Pull request closed without merging**